packages/gatsby/src/commands/develop.js

```diff
@@ -375,7 +375,7 @@ module.exports = async (program: any) => {
     console.log()
     console.log(`Note that the development build is not optimized.`)
     console.log(
-      `To create a production build, use ` + `${chalk.cyan(`gatsby build`)}`
+      `To create a production build, use ` + `${chalk.cyan(`npm run build`)}`
```

Comment:
The commands themselves should probably stay as-is!

Reply:
Woops! Thought I was searching just in the docs but apparently not.

Reply:
Though perhaps this instruction will fail if it remains unchanged? Assuming users probably won't all have the cli globally installed following the new docs.